lib/rubocop/cop/style/each_for_simple_loop.rb

```diff
@@ -28,14 +28,14 @@ class EachForSimpleLoop < Base
         MSG = 'Use `Integer#times` for a simple loop which iterates a fixed number of times.'
 
         def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
-          return unless offending_each_range(node)
+          return unless offended?(node)
```

Comment:
I think `offending?` is a better term here. :)

Reply:
You're right, thanks 😃